packages/@aws-cdk/aws-servicecatalog/test/integ.tag-option.ts

```diff
@@ -0,0 +1,21 @@
+import * as cdk from '@aws-cdk/core';
+import * as servicecatalog from '../lib';
+
+const app = new cdk.App();
+const stack = new cdk.Stack(app, 'integ-servicecatalog-tag-option');
+
+new servicecatalog.TagOption(stack, 'MyTagOption1', {
+  key: 'tagOptionKey',
+  value: 'tagOptionValue',
+});
+
+// const portfolio = new servicecatalog.Portfolio(stack, 'MyPortfolio', {
```

Comment:
whoops, will remove

Reply:
you can't actually use an import in integ test since it will check if the import actually exists.